### DISCUSSION TITLE
Multiple notebook support

### DISCUSSION THREAD

**Discussion by robbienohra** [zk-org/zk#203] [Help] [answered]
Is it possible to have more than one notebook?

**Comment by robbienohra** [comment, marked as answer]
Nvm I see I've answered my own question 😅 
Indeed it looks like there is a natural mapping between multiple configuration files and notebooks:

https://github.com/mickael-menu/zk/blob/main/docs/config.md

**Reply by mickael-menu** [reply]
Right, see this note for more information: https://github.com/mickael-menu/zk/blob/main/docs/notebook.md
